src/dpq2/conv/time.d

```diff
@@ -13,13 +13,13 @@ import dpq2.oids;
 import dpq2.conv.to_d_types: throwTypeComplaint;
 import dpq2.exception;
 
-import std.datetime;
-import std.bitmanip: bigEndianToNative;
+import core.time;
+import std.datetime.date : Date, DateTime, TimeOfDay;
+import std.datetime.systime : SysTime, UTC;
+import std.bitmanip: bigEndianToNative, nativeToBigEndian;
 import std.math;
 import core.stdc.time: time_t;
 
-pure:
```

Comment:
I am loved this line :)

Reply:
I understand :)
But as some methods of SysTime aren't pure I'm torn between solutions.
* I can mark methods individually as pure, but it is error prone and so I don't like it much
* I can reorganize the file a bit so pure methods will be down the file - but then also private methods would be splitted in file between pure/impure sections
* I can move impure functions to something like time_impure.d - but I don't like that much either :)

So I'll keep this one open to see in the end how we'd work it out finally.

Reply:
Pure saved :)
Just one method went up after some added templates reverts